annif/transformer/transformer.py

```diff
@@ -0,0 +1,52 @@
+"""Common functionality for input transforming."""
+
+from annif.corpus import TransformingDocumentCorpus
+from annif.exception import ConfigurationException
+
+
+class IdentityTransformer():
+    """"""""  # TODO
+
+    name = 'pass_through'
```

Comment:
Here it's `pass_through`, but in the `__init__.py` dict it was just `pass`, this should be consistent I think?